husky_directory/templates/base.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="eng">
+<head>
+    <meta content="IE=edge" http-equiv="X-UA-Compatible"/>
+    <title>UW Faculty/Staff/Student Directory</title>
+    <meta http-equiv="content-type" content="text/html;charset=ISO-8859-1" />
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0" />
+    <meta http-equiv="Cache-Control" content="no-cache" />
+    <meta http-equiv="Pragma" content="no-cache" />
+    <meta http-equiv="Expires" content="0" />
+    <meta name="googlebot" content="noarchive" />
+    <meta name="robots" content="noarchive" />
+
+    <!-- Desktop -->
+    <link rel="stylesheet"
+          href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.css"
+          type="text/css"/>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/mini.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/body.css') }}">
+    <link rel="shortcut icon" href="{{ url_for('static', filename='assets/favicon.ico') }}">
+
+    <link rel="stylesheet" type="text/css" media="print" href="{{ url_for('static', filename='css/print.css') }}">
+
+    <style>
+        .btn.search:hover { color: black; }
+        /*------------ Workday changes --------- */
+        ul.dir-listing { list-style-type: none; padding: 0; }
+        ul.multiaddr, ul.title, ul.address { list-style-type: none; padding: 0; }
+        li.dir-boxstuff { padding-bottom: .6em }
+        /*------------ End Workday stuff-------*/
+    </style>
+</head>
+
+<body>
+{% include "header.html" %}
+<div class="container">
+    <div class="usebar">
+        <div class="titleback">
+            <div class="title">
+                <h2 style="margin-top:40px;">UW Directory</h2>
+            </div>
+        </div>
+        {% include "search.html" %}
+        <hr/>
+        {% block content %}{% endblock %}
+        {% include "links.html" %}
+        {% include "footer.html" %}
+    </div>
+</div>
+</body>
+</html>
```

Comment:
Most of this is unchanged, excepting the new `{% block content %}{% endblock %}` which allows us to more easily separate views, changing only the parts of the render we need for the view. I removed some commented cruft also.

Reply:
Does anything provide content for the `{% block content %}{% endblock %}` currently, or is this a placeholder? I don't see changes to anything that uses fill this block, so...placeholder?

Reply:
Nevermind that comment, now that I actually look at the new files/changes, we have content, just more organized. You kept your word!
Oh, and looks like someone had some fun with this :)

Reply:
Don't I always have fun with refactors? 😄